cmd/codebuild/reset/main.go

```diff
@@ -221,7 +221,7 @@ func generateNukeConfig(svc *service, f io.Writer) error {
 		AdminRole:       config.accountAdminRoleName,
 		PrincipalRole:   config.accountPrincipalRoleName,
 		PrincipalPolicy: config.accountPrincipalPolicyName,
-		Regions:         config.allowedRegions,
+		Regions:         config.nukeRegions,
```

Comment:
Renamed to `nukeRegions` (it's not regions we're allowing, it the regions we're nuking)